src/test/java/com/example/CustomerService/services/CustomerLoginServiceTest.java

```diff
@@ -0,0 +1,72 @@
+package com.example.CustomerService.services;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.example.CustomerService.entity.CustomerCredentials;
+import com.example.CustomerService.model.Credentials;
+import com.example.CustomerService.repository.CustomerCredetialsRepository;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class CustomerLoginServiceTest {
+
+@Autowired
+CustomerCredetialsRepository testCustomerCredetialsRepository;
+  @InjectMocks
+CustomerLoginService customerLoginService;
+
+
+
+  @Test
+   void login() {
+    String name="niyati";
+    CustomerCredentials expectedCustomerCredentials= CustomerCredentials.builder().username("niyati")
+        .password("niyati123").build();
+    testCustomerCredetialsRepository.save(expectedCustomerCredentials);
```

Comment:
No need to save it to DB as we have already tested that in repository tests. You can use mockito when then return. Refer to Abhishek PR